src/sqlfluff/rules/L044.py

```diff
@@ -83,7 +83,6 @@ def _analyze_result_columns(self, query: Query):
         # Recursively walk from the given query (select_info_list) to any
         # wildcard columns in the select targets. If every wildcard evdentually
         # resolves to a query without wildcards, all is well. Otherwise, warn.
-        assert query.selectables
```

Comment:
Was there a particular test case causing this assertion to fail? Curious because, as annoying as assertion failures are, it has detected a couple of bugs and may detect future ones. Rather than remove it, perhaps we should log an error so problems don't go undetected?

Reply:
Sorry that's fixed in #2468 not #2470